testground/benchmark/benchmark/transaction.py

```diff
@@ -77,6 +78,8 @@ def load(datadir: Path, global_seq: int) -> [str]:
         return ujson.load(f)
 
 
+@backoff.on_predicate(backoff.expo, max_time=60, max_value=5)
+@backoff.on_exception(backoff.expo, aiohttp.ClientError, max_time=60, max_value=5)
```

Comment:
seems can random_jitter to avoid many requests stuck at the same time
```suggestion
@backoff.on_predicate(
    backoff.expo, max_time=60, max_value=5, jitter=backoff.random_jitter
)
@backoff.on_exception(
    backoff.expo,
    aiohttp.ClientError,
    max_time=60,
    max_value=5,
    jitter=backoff.random_jitter,
)
```

Reply:
the default `full_jitter` makes more sense to me, it's also randomized.